packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-questionnaire-submission/submit/controller.js

```diff
@@ -0,0 +1,10 @@
+const BackOfficeV2Service = require('../../../../../../services/back-office-v2');
+
+const backOfficeV2Service = new BackOfficeV2Service();
+
+/** @type {import('express').Handler} */
+exports.post = async (req, res) => {
+	await backOfficeV2Service.submitQuestionnaire(req.params.caseReference);
```

Comment:
oh, you should probably catch an error here and throw that unable to submit one that used to be in response.service or wherever I left a note for myself

Reply:
did that